src/cli/subcommands/wait_api_cmd.rs

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019-2025 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use crate::rpc::{self, prelude::*};
+use std::time::Duration;
+
+#[derive(Debug, clap::Args)]
+pub struct WaitApiCommand {
+    /// duration to wait till fail
+    #[arg(long, default_value = "30s")]
+    timeout: humantime::Duration,
```

Comment:
I would expect the `forest-cli wait-api` to wait indefinitely by default. How about making it `Option<humantime::duration>`, and, if not provided, making it wait forever?

Reply:
The current default value matches `lotus wait-api`, but of course it can be infinity by default, what do you think?

Reply:
@LesnyRumcajs  Done.

Reply:
Excellent, thanks.